pkgs/clightning-plugins/get-sha256.sh

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p git gnupg
+set -euo pipefail
+
+TMPDIR="$(mktemp -d -p /tmp)"
+trap "rm -rf $TMPDIR" EXIT
+cd $TMPDIR
+
+echo "Fetching latest release"
+git clone https://github.com/lightningd/plugins 2> /dev/null
+cd plugins
+
+latest="$(git rev-parse master)"
+
+echo "ref: ${latest}"
+echo "sha256: $(git archive --format tar.gz --prefix=clightning-plugins-"${latest}"/ ${latest} | sha256sum | cut -d\  -f1)"
```

Comment:
This should either

a) yield two sha256's that can be used with `fetchFromGitHub` for both `mach-nix` and `plugins` 

OR

b) switch `mach-nix` and `plugins` to `fetchurl` in `default.nix` so they can use the sha256 hashes generated by this `get-sha256.sh`.